rank_models/rank_model_cmd_v5.0_single_withoutmodels_onco.ini

```diff
@@ -1,5 +1,5 @@
 [Version]
-  version = 4.2
+  version = 5.0

```

Comment:
Should this one be here?

Reply:
don't know actually. Dont think it matters too much, but it's weird having a filename with version and a header in said file with another version

Reply:
OK I think I'll leave it out of this PR. We can correct it separately if we want